netlogo-gui/resources/system/behaviorspace.dtd

```diff
@@ -11,6 +11,7 @@ section of a NetLogo model file.
 <!ATTLIST experiment
   name CDATA ""
   repetitions CDATA "1"
+  useDefaultRunOrder CDATA #IMPLIED
```

Comment:
I'd like to see the property name here changed from `useDefaultRunOrder` to `sequentialRunOrder` or `useSequentialRunOrder`. Although "Default" is clearer right now, it will be quite unclear in 6 months.